src/Molecules/Footer/Footer.js

```diff
@@ -0,0 +1,35 @@
+import propTypes from 'prop-types';
+import FooterProfileBtn from '../../Atoms/FooterProfileBtn/FooterProfileBtn';
+import { Logo, LogoWrapper, Copy, Wrapper } from './FooterStyled';
+import logoDevsOnTheWaves from '../../Assets/logoDevsOnTheWaves.svg';
+
+const Footer = () => {
+  return (
+    <>
+      <Wrapper>
```

Comment:
maybe you can give more descriptive name than only Wrapper?

Reply:
It is local, so maybe it is not necessary, don't waste time on this alone, we can talk in team and choose one way of naming

Reply:
FooterWrapper is ok?